spec/facter/facter_spec.rb

```diff
@@ -435,6 +435,25 @@ def mock_collection(method, os_name = nil, error = nil)
       end
     end
 
+    describe '#load_external' do
+      before do
```

Comment:
probably we should add a test when the external facts are not loaded

Reply:
There already is an acceptance test added with both scenarios.